hknweb/candidate/views.py

```diff
@@ -234,10 +236,32 @@ def get_rand_photo(width=400):
         urls = f.readlines()
     return urls[randint(0, len(urls) - 1)].strip() + "?w=" + str(width)
 
-#Takes in all confirmed rsvps and sorts them into types, current hard coded
+def send_cand_confirm_email(request, challenge, confirmed):
+    subject = 'Your Officer Challenge Was Reviewed'
```

Comment:
I Have Opinions About The Use Of Capitalization In Email Titles. This Feels Too Long To Capitalize, I Think
```
subject = '[HKN] Your officer challenge was reviewed!'
```
looks better.

Reply:
So the other subject is fine? 'Confirm Officer Challenge'